faq.html

```diff
@@ -296,8 +319,6 @@ <h3>Can I receive sats through keysend?</h3>
   <section id="connect-bitcoin-node">
     <h3>How do I connect Blixt Wallet to my own full node?</h3>
     <p>Under <i>Settings -> Bitcoin Node</i>, you can enter a Bitcoin node to connect to.</p>
-    <p>BIP157 compact filter indexing must be enabled for your node. Add this to your bitcoin.conf if it isn't already
```

Comment:
I don't think this should be removed?

Reply:
I have a conflict with master, that's why I remove it. Wanted to add it later after a merge